packages/protocol/migrationsConfig.js

```diff
@@ -64,6 +64,7 @@ const DefaultConfig = {
     targetVotingGoldFraction: 2 / 3,
     maxValidatorEpochPayment: '205479452054794520547', // (75,000 / 365) * 10 ^ 18
     communityRewardFraction: 1 / 4,
+    // TODO(asa): Must be set before RC1
```

Comment:
There shouldn't be any urgency in setting this before RC1.  If its 0, no transaction will occur
https://github.com/celo-org/celo-blockchain/blob/dfd70edf41618d6b40aafd695087d0cf919e0f92/consensus/istanbul/backend/pos.go#L119

Reply:
Nice